envy.go

```diff
@@ -33,6 +39,53 @@ func init() {
 	}
 }
 
+// Reload the ENV variables. Useful if
+// an external ENV manager has been used
+func Reload() {
+	env = map[string]string{}
+	loadEnv()
+}
+
+// Load .env files. Files will be loaded in the same order that are received.
+// Redefined vars will override previously existing values.
+// IE: envy.Load(".env", "test_env/.env") will result in DIR=test_env
+// If no arg passed, it will try to load a .env file.
+func Load(files ...string) error {
+
+	// If no files received, load the default one
+	if len(files) == 0 {
+		err := godotenv.Overload()
+		Reload()
```

Comment:
Then we wouldn't be returning the error.

We could change it to

```go
err := godotenv.Overload()
if err == nil {
  Reload()
}
return err
```

to avoid the unnecessary `Reload` call (and indeed returning the error)